WordPress/src/main/java/org/wordpress/android/models/MenuModel.java

```diff
@@ -20,6 +20,10 @@
     public List<MenuLocationModel> locations;
     public List<MenuItemModel> menuItems;
 
+    public static final long ADD_MENU_ID = -1;
```

Comment:
moved these constants from MenuLocationModel to MenuModel as they seemed to belong here